src/app/CheckFeedLicenses.js

```diff
@@ -0,0 +1,7 @@
+const CheckFeedLicenses = {
+  ACADEMIC: 1,
+  COMMERCIAL: 2,
+  OPEN_SOURCE: 3,
+};
+
+export default CheckFeedLicenses;
```

Comment:
@amoedoamorim do you need anything similar for your code?

Reply:
anyway, I'll wait for you to merge first

Reply:
Yes, sounds good!